android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "appId": "app",
+  "appId": "cha",
```

Comment:
I think the app id here should be `app` instead of `cha`

Reply:
agree
```suggestion
  "appId": "app",
```